### PR TITLE
Changed Docker file to be albe to auto build image by dockerhub

### PR DESCRIPTION
**Pull request by ifraixedes**
I've changed Dockerfile and makefile targets accordingly to be
able to auto build the image by dockerhub.

I also added "test" target which only run the image creaing to queues

@mwagg @croman @pilu please review this

### CODE REVIEW

**Comment by mwagg** [comment]
What's your plan for getting this into the registry? We build it locally and push?

**Comment by ifraixedes** [comment]
I've created this PR because after I had a quick read to dockerhub docs (to refresh my mind) I read that Dockerfile must be in the repository root level if you want to have automatic builds based in pushes to the source repository

**Comment by mwagg** [comment]
So at the moment the docker hub no longer works with github such that we can add new automated build. We could manually push it though, which may be fine in this case

**Comment by ifraixedes** [comment]
1. Really? I didn't know that, probably I've read some old docs.
2. We could do it with this too. I don't have any problem in doing it in that way, automatic builds is better to avoid that step each time that we update but it's totally fine doing manually.

**Comment by mwagg** [comment]
It should work, but doesn't. They don't see any repositories owned by us. It's because they don't use the new GitHub third party auth stuff I think

**Comment by ifraixedes** [comment]
:+1:  thanks for the info.

I'm not sure if you're waiting for my green light to push it as I manifested that I'm happy with it, but just in case that you're waiting for it, here it's my green light.

I also want my green light for this :smile:

**Comment by mwagg** [comment]
How do you use it to test it?

You should add an example to the readme

**Comment by ifraixedes** [comment]
1. For now the only test is that you run `make dev-env test` and you get two worthless messages in the terminal saying "Create queue test1" and "Creating queue test2" and any message that something has gone wrong. Totally useless for test purpose, I know.
   I'm using in our internal project that needed this functionality and it worked, but for the public world isn't worth it.
2. Yeah, I know, we have been busy in DIGIT stuff.
   It's on the plan for that reason I added a [TODO](https://github.com/DreamItGetIT/sqs-initialiser#todo) in README but I haven't got so far a spot in my spare time to do it.

File: makefile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .PHONY: build
 
+DOCKER_IP=$(shell ip route | grep docker0 | awk '{ print $$9 }')
```
Comment (croman) [inline makefile]: You should add here the `grep src` like we used in other places.

**Comment by croman** [comment]
One small comment, but looks good!
https://i.chzbgr.com/full/8573958656/h552A4FB8/

**Pull request merged**